data/json/items/comestibles/carnivore.json

```diff
@@ -43,7 +43,7 @@
     "type": "COMESTIBLE",
     "name": "human stomach",
     "description": "The stomach of a human.  It is surprisingly durable.",
-    "material": "hflesh"
```

Comment:
why are you removing the tag that says this human stomach is made of human flesh?

Reply:
Accident Ill fix it and reupload